src/bullets/__init__.py

```diff
@@ -1,6 +1,4 @@
-from .portfolio import *
-from .runner import *
-from .strategy import *
+from bullets import *
```

Comment:
This is not good practice in Python. We only we want essential classes here to make them easier to import in our code.

This is a good explanation: https://stackoverflow.com/questions/61801276/best-practices-for-top-level-init-py-imports

Reply:
fixed